### ISSUE TITLE
Variables in the size function causes an error.

### ISSUE THREAD

**Issue by zachooz** [processing-js/processing-js#144]
When one puts variables in the size function, it works in java mode but not in processing js.

For Example
```
int x = 500;
int y = 500;

void setup(){
  size(x,y);
  background(255,0,0);
}
```

Throws this error
```
Exception in thread "Processing.BasicServer" java.lang.NoSuchMethodError: processing.app.Base.showWarning(Ljava/lang/String;Ljava/lang/String;Ljava/lang/Exception;)V
	at de.bezier.mode.javascript.JavaScriptBuild.build(Unknown Source)
	at de.bezier.mode.javascript.JavaScriptBuild.export(Unknown Source)
	at de.bezier.mode.javascript.JavaScriptMode.handleExport(Unknown Source)
	at de.bezier.mode.javascript.JavaScriptEditor.handleExport(Unknown Source)
	at de.bezier.mode.javascript.JavaScriptEditor.serverStarted(Unknown Source)
	at de.bezier.mode.javascript.BasicServer.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
```

When put online, all that appears is a black box

**Comment by Pomax** [comment]
you want https://github.com/processing/processing/issues for that, since that's a Processing issue (not Processing.js), but they'll tell you this is because size() *cannot* take values that might change at some point, so you can either hardcode the values as `size(500,500); x = width; y = height;`, or you have to make your ints constants, using `final int x = 500, y = 500`.

**Comment by GoToLoop** [comment]
I've got some codes which calculates _width_ & _height_ for **size()** and they sorta work in PJS.
You can view it online too: http://studio.processingtogether.com/sp/pad/export/ro.98-pSvW97Q-RE/latest

```processing
/**
 * GridGUI Sample (v2.01)
 * by GoToLoop (2013/May)
 *
 * forum.processing.org/topic/basic-gui
 * studio.processingtogether.com/sp/pad/export/ro.98-pSvW97Q-RE/latest
 */

static final byte GRID = 8;
boolean[][] leds = new boolean[GRID][GRID];

static final color OFF = 0350, ON  = #0000FF;
static final short DIM = 0100;

void setup() {
  size(GRID*DIM + 1, GRID*DIM + 1, JAVA2D);
  noLoop();
  frameRate(60);

  smooth(4);
  stroke(0);
  strokeWeight(1.5);

  initGrid();
}

void draw() {
  for (int row = 0; row != GRID; ++row)
    for (int col = 0; col != GRID; ++col) {
      fill(leds[row][col] ? ON : OFF);
      rect(col*DIM, row*DIM, DIM, DIM);
    }
}

void keyPressed() {
  initGrid();
  redraw();
}

void mousePressed() {
  keyPressed();
}

void initGrid() {
  for (int row = 0; row != GRID; ++row)
    for (int col = 0; col != GRID; ++col)
      leds[row][col] = random(1) > .5;
}
```

**Comment by fjenett** [comment]
This is fixed in JS mode as of version 0.3.2 … please update (remove / reinstall) in Processing PDE.
